src/CssInliner.php

```diff
@@ -93,22 +93,22 @@ class CssInliner extends AbstractHtmlProcessor
     ];
 
     /**
-     * @var CssSelectorConverter
+     * @var CssSelectorConverter|null
```

Comment:
Is `@var ?CssSelectorConverter` possible/preferable?

Reply:
Yes, changed.